### ISSUE TITLE
NixError on MacOS: dyld: malformed mach-o image

### ISSUE THREAD

**Issue by chrismaree** [dapphub/dapptools#383]
Hi! I've been having a lot of trouble getting the dapphubs took kit installed. I have tried both the easy install and the Manual install from the repo but always get this error with seth:
```
seth ls
dyld: malformed mach-o image: segment __DWARF has vmsize < filesize

```

I have googled it to the end of the earth and the only thing I really found was a connection to a Go error on a specific version. I've tried updating my go but still cant get past this. any help would be super helpful.

I'm on MacOS Catalina & I have Go installed via home brew.

Thanks!

**Comment by xwvvvvwx** [comment]
Do you only see this error with `seth ls` or does it occur with all `seth` commands?

**Comment by chrismaree** [comment]
> Do you only see this error with `seth ls` or does it occur with all `seth` commands?

I saw this with a bunch of commands. `seth send` for example did the same thing. I was able to run `seth call` without this error occurring.

**Comment by xwvvvvwx** [comment]
interesting....

Could you try running this:

```
nix run -f $DAPPTOOLS ethsign --command ethsign
```

**Comment by chrismaree** [comment]
> interesting....
> 
> Could you try running this:
> 
> ```
> nix run -f $DAPPTOOLS ethsign --command ethsign
> ```

that env variable is not set on my machine so I got:
```
error: getting status of '/Users/chris/ethsign': No such file or directory
```

I was able to simply run `ethsign --command ethsign` and got the same error:
```
ethsign --command ethsign
dyld: malformed mach-o image: segment __DWARF has vmsize < filesize
[1]    45792 abort      ethsign --command ethsign
```

**Comment by MrChico** [comment]
@xwvvvvwx this is some obscure golang error which doesn't exist on the `symbolic` branch. I think bumping nixpkgs and factoring out the changes that are made to ethsign in that branch is the key here

**Comment by MrChico** [comment]
fixed by #384
